recipes/prgenv-gnu/24.11/gh200/environments.yaml

```diff
@@ -0,0 +1,46 @@
+gcc-env:
+  compiler:
+      - toolchain: gcc
+        spec: gcc
+  mpi:
+      spec: cray-mpich@8.1.30
+      gpu: cuda
+  unify: true
+  specs:
+  - boost +chrono +filesystem +iostreams +mpi +python +regex +serialization +shared +system +timer
+  - cmake
+  - fftw
+  - fmt
+  - gsl
+  - hdf5+hl
```

Comment:
Suggestion:
```
hdf5+hl+fortran
```
Unfortunately way too many people use fortran, and why not build the fortran bindings? Applies to configs.

Reply:
Sounds reasonable. What about C++ support? The variant seems to be disabled by default.

Reply:
I do not see any downsides of enabling them too. Fortran was more important for me, because without the bindings, you cannot use the library with your fortran code. Without the C++ bindings, you could still fall back to the C API. Also in my experience, the C++ interface is not used very frequently (which does not mean, that we should disable it by default). I think we should give users the choice to use any interface they prefer.

Reply:
Yep, I don't see any particular reason to leave them out either, except if they were to cause endless build issues, but that seems like an issue we can deal with when/if it happens.

Reply:
Fortran and C++ support enabled in https://github.com/eth-cscs/alps-uenv/pull/159/commits/9091bc7addefd5a6d14109cb30791fa400cb77c7 and https://github.com/eth-cscs/alps-uenv/pull/159/commits/70f3d7c090773c65c83f47e3dc1cc13aafe5b2db, respectively.